src/js/visual/image.js

```diff
@@ -160,9 +289,53 @@ ripe.Image.prototype.update = async function(state, options = {}) {
         size: size,
         width: width,
         height: height,
+        rotation: this.rotation,
         crop: crop,
         initials: initialsSpec.initials,
-        profile: initialsSpec.profile
+        profile: initialsSpec.profile,
+        flip: this.flip,

```

Comment:
`flip: flip` so that it is aligned with how `crop` works.

Reply:
Not sure if I understood